app/tray_icon.js

```diff
@@ -0,0 +1,96 @@
+const electron = require('electron')
+const path = require('path');
+
+const app = electron.app;
+const Menu = electron.Menu;
+const Tray = electron.Tray;
+
+let trayContextMenu = null;
+let tray = null;
+
+function createTrayIcon(mainWindow, messages) {
+
+  // TODO: for better desktop theme integration, it would be good to provide
+  // multiple tray icons (light, dark, colour) and allow selection from the options
+  tray = new Tray(path.join(__dirname, '../images/icon_48.png'));
```

Comment:
How about `path.join(__dirname, '..', 'images', 'icon_48.png')` instead? Keep those pesky linux-y `/` characters out of a cross-platform codebase? :0)